readme.md

```diff
@@ -111,6 +111,79 @@ python3 -m nwm_routing -f -V4 test_AnA_V4_HYFeature.yaml
 
 **Note**: The following instructions are for setting up T-Route on a Linux environment (standalone, no MPI). If you are using Windows, please install WSL (Windows Subsystem for Linux) before proceeding.
 
+### T-Route Setup and Testing Guide for Windows Users WITHOUT conda [based on pip and venv - only widely available dependencies]
+
+1. **Install Recommended distro:**
+   - Download and install WSL2 for your Windows OS
+   - We recommend long-term stable (LTS) Ubuntu distribution 22.04 or 20.04 (24.04 not recommended yet)
+   - Open Windows Power Shell and issue
+     > wsl --install Ubuntu-22.04
+   - Enter (root) username and password (2x) of your choice
+     
+2. **Set up venv-based virtual environment:**
+   - From root (the username you created under 1):
+      - Update Linux distro:
+         > sudo apt update
+      - Install pip (package manager):
+         > sudo apt install python3-pip
+      - Install venv:
+         > sudo apt install python3.10-venv
```

Comment:
I think up to this part is for first time user. If they installed t-route before and they need new clone. These steps are not required.

Reply:
The instructions are meant for a new user to setup a pip-based environment based on a Ubuntu distro in WSL. Neither pip nor venv come with the standard distro, therefore they need to be installed - if I understand the objection right?

Reply:
Instructions are meant to cover installation on a new Ubuntu distro. For updating an existing distro, we can, at some point, set up a separate instruction set (or even an "update script"). But for the time, we are paralleling the conda based instructions. 

I will, though, include a warning not to install this within an existing conda virtual environment, which might break!